index.html

```diff
@@ -1,14 +1,161 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="style.css">
-  <title>Free thinking Conference: The value of reason, logic, and evidence.</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+  <title>Free Thinking Conference: The value of reason, logic, and evidence.</title>
 </head>
-<body>
-  <h1>hello world!</h1>
+
+<body class="body">
+  <div class="mobile-menu hide">
+    <img class="mobile-menu-close" src="images/icons/pop-close.svg" alt="close">
+    <ul class="mobile-menu-elements">
+      <li class="mobile-menu-elements-element">Program</li>
+      
+      <li class="mobile-menu-elements-element" id="about-link"><a href="#about">About</a></li>
+      <li class="mobile-menu-elements-element">Join</li>
+      <li class="mobile-menu-elements-element">Sponsor</li>
+      <li class="mobile-menu-elements-element">News</li>
+      <li class="mobile-menu-elements-element">CC Campaign</li>
+    </ul>
+  </div>
```

Comment:
- [ ] Dear @danifromecuador , Please notice the nav bar is divided into a top dark bar and a bottom white bar on the original design. I notice you forget the top short bar section. Please also make sure to add social media links, log out, and language links aligned to the right.
### Original design
![image](https://user-images.githubusercontent.com/69077061/225538557-5fda6188-296b-4fb7-823d-205d0ae0502c.png)